src/toil/batchSystems/mesos/batchSystem.py

```diff
@@ -572,6 +572,12 @@ def executorLost(self, driver, executorId, slaveId, status):
         log.warning("Executor '%s' lost.", executorId)
 
 
+    @classmethod
+    def setOptions(cl, config):
+        super.setOptions(config)
```

Comment:
I think this should be `super(MesosBatchSystem, self).setOptions(config)` - that seems to be part of the reason the jenkins build is failing